packages/datastore/datastore/src/ModelSchema.ts

```diff
@@ -14,28 +14,75 @@ export interface GenetratedModelSchema {
 }
 
 export interface ModelSchemaProperties extends JSONSchema7 {
+  /** Index field, whether or not the
+   * field should be indexed
+   */
   index?: boolean;
+  /** Primary key field flag if the field is
+   * the primary key
+   */
   primary?: boolean;
+  /**
+   * Default value for the field
+   */
   default?: any;
+  /** Flag for if the field should be encrypted
+   * i.e. password field
```

Comment:
```suggestion
   * i.e. password field
```

Password field should never be passed here :D

Reply:
@wtrocki so the value will never be passed there, but surely at some point the model will have to handle sensitive fields that might need encryption, or not really?

Reply:
Yep. Just remove password as example.